tests/ui/consts/const-fn-in-vec.rs

```diff
@@ -1,7 +1,11 @@
+static _MAYBE_STRINGS: [Option<String>; 5] = [None; 5];
```

Comment:
If we annotate this file with `// run-rustfix`, does the test continue to pass? If so, let's add that to protect against involuntary regressions in the suggestions. (You might have to `allow` some lints, potentially.)

Reply:
Ah unfortunately no, because the const uses the same name which conflicts. It might be a good idea to maybe make the identifier more unique (do we have a way of 'identifier-izing' an arbritrary string?). Something like `Enum::Variant` to `ENUM_VARIANT` might help